plugins/org.locationtech.udig.catalog.ui/src/org/locationtech/udig/catalog/ui/export/CatalogExportWizard.java

```diff
@@ -294,6 +296,31 @@ public void run() {
         return true;
     }
 
+    private SimpleFeatureType removeUnsupportedTypes(final SimpleFeatureType destinationFeatureType) {
```

Comment:
Aha, looks like removal of special (generic) attributes comming from feature-type. Attributes with type Object are ignored and a message is sent to System.out

I would appreciate it to log a warning message (wich would occur in errorlog-view. A comment for this private method would be great too :)

Is it possible to provide a testcase for such a constellation? What would happen if this methoud would not exists? Is the export broken?